lib/services.js

```diff
@@ -450,6 +450,7 @@ export default {
             metadata.putObjectMD(bucketName, longMPUIdentifier,
                 multipartObjectMD, log, (err) => {
                     if (err) {
+                        log.error('error from metadata', { error: err });
```

Comment:
I know this may sound like nitpicking, but are we sure we won't be missing error cases this way ? If yes, then fine with me.